staging/src/k8s.io/client-go/tools/cache/controller.go

```diff
@@ -143,6 +144,9 @@ func (c *controller) Run(stopCh <-chan struct{}) {
 	if c.config.WatchErrorHandler != nil {
 		r.watchErrorHandler = c.config.WatchErrorHandler
 	}
+	if s := os.Getenv("ENABLE_CLIENT_GO_WATCH_LIST"); len(s) > 0 {
```

Comment:
I would really like to avoid controlling that with an env-var.
We should use an explicit parameter for that.